extensions/sql-bindings/src/test/common/azureFunctionsUtils.test.ts

```diff
@@ -0,0 +1,103 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import * as vscode from 'vscode';
+import * as path from 'path';
+import * as fs from 'fs';
+import * as should from 'should';
+import * as sinon from 'sinon';
+import * as constants from '../../common/constants';
+import * as azureFunctionsUtils from '../../common/azureFunctionsUtils';
+
+let rootFolderPath = 'test';
+let localSettingsPath: string = 'test/local.settings.json';
+let projectFilePath: string = 'test/projectFilePath.csproj';
+
+describe('Tests to verify Azure Functions Utils functions', function (): void {
+
+	it('Should get local.settings.json', async () => {
+		sinon.stub(fs, 'existsSync').withArgs(localSettingsPath).returns(true);
+		sinon.stub(fs, 'readFileSync').withArgs(localSettingsPath).returns(
+			`{"IsEncrypted": false,
+			"Values": {"test1": "test1", "test2": "test2", "test3":"test3"}}`
+		).toString();
+		let settings = await azureFunctionsUtils.getLocalSettingsJson(localSettingsPath);
+		should(settings.IsEncrypted).equals(false);
+		should(Object.keys(settings.Values!).length).equals(3);
+	});
+
+	it('Should show error message if local.settings.json throws', async () => {
+		let errorMsg = 'Error parsing local.settings.json';
+		sinon.stub(fs, 'existsSync').withArgs(localSettingsPath).returns(true);
+		sinon.stub(fs, 'readFileSync').withArgs(localSettingsPath).returns(
+			`{"IsEncrypted": false,
+			"Values": {"test1": "test1", "test2": "test2", "test3":"test3"}}`
+		).toString();
+		const getLocalSettingsSpy = sinon.spy(azureFunctionsUtils, 'getLocalSettingsJson');
+		sinon.stub(JSON, 'parse').withArgs(sinon.match.any).throws(new Error(errorMsg));
+		try {
+			await getLocalSettingsSpy(localSettingsPath);
```

Comment:
If all you're doing is checking that `getLocalSettingsJson` threw then you can just use `should(() => function()).throwError()`